client/src/components/DisplayTable/DisplayTable.tsx

```diff
@@ -520,50 +574,56 @@ export class DisplayTable extends React.Component {
               <NoDataMessage />
             ) : (
               <tbody>
-                {_.map(paginated_data[current_page], (row, row_index) => (
-                  <tr key={row_index}>
-                    {_.map(visible_ordered_col_keys, (col_key, col_index) => (
-                      <td
-                        style={{
-                          fontSize: "14px",
-                          textAlign: determine_text_align(row, col_key),
-                        }}
-                        key={col_key}
-                        tabIndex={-1}
-                        ref={
-                          col_index === 0 && row_index === 0
-                            ? this.first_data_ref
-                            : null
-                        }
-                      >
-                        {col_configs_with_defaults[col_key].formatter ? (
-                          _.isString(
-                            col_configs_with_defaults[col_key].formatter
-                          ) ? (
-                            <Format
-                              type={
+                {_.map(
+                  paginated_data[current_page],
+                  (row: DisplayTableData, row_index) => (
+                    <tr key={row_index}>
+                      {_.map(
+                        visible_ordered_col_keys,
+                        (col_key: string, col_index) => (
+                          <td
+                            style={{
+                              fontSize: "14px",
+                              textAlign: determine_text_align(row, col_key),
+                            }}
+                            key={col_key}
+                            tabIndex={-1}
+                            ref={
+                              col_index === 0 && row_index === 0
+                                ? this.first_data_ref
+                                : null
+                            }
+                          >
+                            {col_configs_with_defaults[col_key].formatter ? (
+                              _.isString(
                                 col_configs_with_defaults[col_key].formatter
-                              }
-                              content={row[col_key]}
-                            />
-                          ) : (
-                            col_configs_with_defaults[col_key].formatter(
+                              ) ? (
+                                <Format
+                                  type={
+                                    col_configs_with_defaults[col_key].formatter
+                                  }
+                                  content={row[col_key]}
+                                />
+                              ) : (
+                                col_configs_with_defaults[col_key].formatter(
```

Comment:
Like Garvin was saying, getting `  No constituent of type 'string | Function' is callable` here